packages/domain/src/use-cases/publications/__helpers__/mocks.ts

```diff
@@ -133,7 +133,7 @@ export function mockCreateMirrorRequest(
   return {
     profileId: mockProfileId(),
     publicationId: faker.datatype.uuid(),
-    publicationType: PublicationType.COMMENT,
+    reference: ReferencePolicy.ANYBODY,
```

Comment:
Although supported at protocol level, adding reference module here might promote behaviour that dilutes conversations in mirror sub-threads?

This worth a conversation.

Reply:
agree, will remove to avoid confusion.

Reply:
Done